src/services/youtube.ts

```
@@ -52,4 +70,117 @@ export class YouTubeService extends BaseClient {
+      if (limit !== undefined) {
+        params.limit = limit;
+      }
+      const response = await this.fetch<YoutubeVideoList>('/youtube/channel/videos', params);
+      return response.videoIds;
+    }
+  }
+
+  export class Playlist extends BaseClient {
+    constructor(config: SupadataConfig) {
+      super(config);
+    }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)